core/data/src/androidMain/kotlin/io/github/droidkaigi/confsched2023/data/di/ApiModule.kt

```diff
@@ -159,6 +160,8 @@ class RemoteConfigModule {
     @Provides
     @Singleton
     fun provideFirebaseRemoteConfig(): RemoteConfigApi {
-        return DefaultRemoteConfigApi()
```

Comment:
We can move DefaultRemoteConfigApi to androidMain and you can leave this as it is

Reply:
I was able to confirm the operation and get a normal value.

07e9400877a2c4afc63436ba3f73a0c299db6a3e

Reply:
It's not for Andorid but for iOS 🙏 
When iOS use DefaultRemoteConfigApi, DefaultRemoteConfigApi don't have the implementation class right?
Can you check this?
./gradlew :app-ios-shared:assembleSharedReleaseXCFramework